tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -242,6 +332,26 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 	})
 })
 
+func CheckPeerMembersAndClusterStatus(clusterCli ctrlCli.Client, tc *v1alpha1.TidbCluster, expectNonExistedTc *v1alpha1.TidbCluster) error {
+	clusterCli.Get(context.TODO(), types.NamespacedName{Namespace: tc.Namespace, Name: tc.Name}, tc)
+
+	if tc.Status.PD.Phase != v1alpha1.NormalPhase || tc.Status.TiKV.Phase != v1alpha1.NormalPhase || tc.Status.TiDB.Phase != v1alpha1.NormalPhase || tc.Status.TiFlash.Phase != v1alpha1.NormalPhase {
```

Comment:
What about CDC and Pump?

Reply:
Fixed, I added these components status check.